### ISSUE TITLE
346348997 HARD WORK ()

### ISSUE THREAD

**Issue by FreeREGcomputer** [FreeUKGen/FreeBMD2#1229]
Issue reported by **** at 2025-12-22 16:03:17 UTC
Time: 2025-12-22T15:43:14+00:00
Session ID: ac88c627c4be0647ea87cf6616835f42
Problem Page URL: [/search_queries/69496502c233301cd02706e9?locale=en&page=6](/search_queries/69496502c233301cd02706e9?locale=en&page=6)
Previous Page URL: [https://www.freebmd2.org.uk/search_queries/69496502c233301cd02706e9?locale=en&page=5](https://www.freebmd2.org.uk/search_queries/69496502c233301cd02706e9?locale=en&page=5)
Reported Issue:
The website is not intuitive it is much too difficult to negotiate. Why say "see possible spouses" and there is nothing to click on to let you view a possible spouse and let you simply return to the list of names displayed from the search? It is far too difficult. It should have been designed by a real person and not an anorak.  The dropdown arrows for "select feedback" are barely visible. A total redesign is needed from a partially sighted person's point of view.    

![Screenshot](https://www.freebmd2.org.uk)

**Comment by nurifreembd** [comment]
All,

Observation on leave feedback form:

1. attached picture on Iphone 13.
1.1. please correct the size and positioning of the form so its not
truncated
1.2 extend the length of the 'select the fee...
1.3 remove underline below the asterisks, not needed unless there is a
valid reason
1.4 green background for input field is not a great contrast.  Consider
white or the same background as the search options.
1.5 description could read ' Feedback Description' and 'enter your
feedback' instead
1.6 label should align with the input field
1.7 page title reads beta... is it still in beta phase now that it is live
- if no, could read' provide feedback'?

[image: Screenshot 2026-01-27 at 17.49.13.png]

**Comment by DeniseColbert** [comment]
@Vino-S to tidy up mobile experience of using the feedback form. Nuri happy to test.

**Comment by stephencoward-source** [comment]
This is the contact form

Nuri to test - 26/5

**Comment by stephencoward-source** [comment]
Nuri - tested - Complete
